README.md

```diff
@@ -93,4 +94,4 @@ Share your apps here! send a pull request!
 * [Twitter](https://twitter.com/appwrite_io) (Official, global)
 * [Facebook](https://www.facebook.com/appwrite.io/) (Official, global)
 * [Appwrite Developers on Facebook](https://www.facebook.com/groups/appwrite.developers/?source_id=227046654430085) (Official, global)
-* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
\ No newline at end of file
+* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
```

Comment:
There is one line break character at it isn't visible as its just a \n try using a code editor like atom, sublime or vscode then you will see a blank line at end you can then remove that line

Reply:
Nope, still not visible. I think it's showing because I have added a line in between so last line is shifted below.

Reply:
![2020-10-01_13-13](https://user-images.githubusercontent.com/52061363/94781793-f48ad500-03e7-11eb-976a-6d42b0408f29.png)

I tried cloning your repo and found it's there you just have to remove the line.

Reply:
yeah got it thanks, and also just updated the file. Now it's fine.